src/sentry/search/events/datasets/discover.py

```diff
@@ -1729,6 +1737,33 @@ def _resolve_web_vital_opportunity_score_function(
             alias,
         )
 
+    def _resolve_count_scores_function(self, args: Mapping[str, Column], alias: str) -> SelectType:
+        column = args["column"]
+
+        if column.key not in [
+            "score.total",
+            "score.lcp",
+            "score.fcp",
+            "score.fid",
+            "score.cls",
+            "score.ttfb",
+        ]:
+            raise InvalidSearchQuery("count_scores only supports performance score measurements")
+
+        return Function(
+            "countIf",
+            [
+                Function(
+                    "greaterOrEquals",
+                    [
+                        column,
+                        0,
+                    ],
```

Comment:
Could we use `isNotNull`? Both work, it's just that seeing `greaterOrEquals` makes me think of what the other option is, which in this case leads me to think there may be negative values?

Reply:
Nice, I didn't know `isNotNull` existed, updated!